public/static/app/perf/TestSample.js

```diff
@@ -59,27 +62,30 @@ mciModule.factory('TestSample', function() {
       return result;
     }
 
-    this.resultForTest = function(testName){
+    this.resultForTest = function (testName) {
       return _.findWhere(
-        this.sample.data.results, {name: testName}
+        this.sample.data.results, {
+          name: testName
+        }
       );
     }
 
-    this.maxThroughputForTest = function(testName, metric){
-      if(!_.has(this._maxes, testName)){
-        var d = this.resultForTest(testName);
-        if(!d){
-          return;
-        }
-        this._maxes[testName] = _.max(
+    this.maxThroughputForTest = function (testName, metric, threadLevel) {
```

Comment:
I suspect that this should return null or undefined for no value. And the calling code should handle null as no results.

The max value could actually be 0.

Reply:
fixed